Expensify/App
# Stop "retrying" requests that fail in the main queue

### Details

Holding on:
- [ ] https://github.com/Expensify/Expensify/issues/208280
- [ ] https://github.com/Expensify/Expensify/issues/208185

### Fixed Issues (Related to)
https://github.com/Expensify/Expensify/issues/208281

### Tests

1. Block a the `UpdateLastRead` request that is triggered whenever a report is switched to (see tip [here](https://umaar.com/dev-tips/68-block-requests/))
1. Navigate to a report while online
1. Verify that the request is not getting retried at all and doesn't throw any errors in the JS console 


### PR Review Checklist

<details>
<summary><h4>PR Reviewer Checklist</h4></summary>


</details>

### QA Steps

Same as Tests


### Screenshots
❌
